test/hackathon_results.py

```diff
@@ -0,0 +1,133 @@
+from sklearn.pipeline import Pipeline
```

Comment:
We'll need to clean this file up some before checkin - I can work on that when I have some free cycles